docs/UserGuide.md

```diff
@@ -270,22 +273,19 @@ View details associated with a client, such as the client's name and phone numbe
 
 Format: `viewClient i/INDEX`
 
-* Display information about the client at the specific `INDEX`.
-* The index refers to the index number shown in the displayed clients' list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* Display information about the client at the specific [`INDEX`](#index) in the client list.
+
 
 Examples:
 * `viewClient i/1`
 
 #### 5.1.4 Delete client: `delClient`
 
-Delete the specified client from MyInsuRec.
+Delete the specified client from *MyInsuRec*.
```

Comment:
`*MyInsuRec*` should probably be change to `_MyInsuRec_`

Reply:
Just wanna check is it in italic? and also why are the 2 asterisks at the start and end?

Reply:
Actually this is italics, both works fine